projects/ngneat/reactive-forms/src/lib/types.ts

```diff
@@ -62,11 +62,60 @@ export type KeyValueControls<T extends Obj> = {
   [K in keyof T]: T[K] extends FormControl<T[K]>
```

Comment:
does `KeyValueControls` still in use here? or we can delete it?

Reply:
It's not. Removed.

Reply:
You're right, it's not being used.
Removed.